Update README.md

innatepanda opened the pull request:
tic tac toe in java

innatepanda committed:
Update README.md

tic tac toe in java

Aayushi-Mittal commented on README.md:
```diff
@@ -45,6 +45,14 @@
+<h2>Java</h2>
+| S. No. 	| Name                 	| Link                                                	| Level                                                   	|
+|:------:	|:--------------------:	|:---------------------------------------------------:	|:-------------------------------------------------------:	|
+:heavy_plus_sign: Add tic tac toe in Java: https://www.youtube.com/watch?v=gQb3dE-y1S4 <img src="https://img.shields.io/badge/-easy-success">
```
@innatepanda Please correct the markdown syntax in line 53 else is fine. 

It should be like this :
`
| 1. | Tic Tac Toe | https://www.youtube.com/watch?v=gQb3dE-y1S4  | <img src="https://img.shields.io/badge/-easy-success"> |
`

innatepanda committed:
Update README.md

(sorry, I got confused with the format mentioned in the contribution.md. Will keep in mind!)

innatepanda committed:
Merge pull request #1 from innatepanda/innatepanda-patch-1

Update README.md

Aayushi-Mittal committed:
Added a new line

Added a new line after line-50 i.e, <h2>Java</h2> so that the data becomes organised in a table. A new line is mandatory after the end and before starting of a table in markdown syntax.

Aayushi-Mittal commented on README.md:
```diff
@@ -45,6 +45,15 @@
+
+
+<h2>Java</h2>
+
```
Added a new line here so that data becomes organised in a table.

Aayushi-Mittal commented:
Thank you for your contribution :)
